spec/framework/core/options/option_store_spec.rb

```diff
@@ -307,4 +307,18 @@
         .from(true).to(false)
     end
   end
+
+  describe '#reset' do
+    it 'resets to default' do
+      default = option_store.all
+
+      option_store.show_legacy = true
```

Comment:
I think this line is not necessary.

Reply:
Good catch, I wanted to change the default values so the test will catch resetting the values.